src/pages/editor/ChannelClaimTableRow.tsx

```diff
@@ -41,67 +56,108 @@ export const ChannelClaimTableRow = ({ claimedChannel, channelType }: ChannelCla
+            }}
+            label={t(`editor.institution.channel_claims.access_policies.${editingPolicy}`)}
+            icon={
+              editingPolicy === 'Everyone' ? <LockOpenIcon fontSize="small" /> : <LockOutlineIcon fontSize="small" />
+            }
+          />
+        </StyledTableCell>
+        <StyledTableCell sx={{ display: 'flex', flexWrap: 'wrap', gap: '0.25rem' }}>
+          {claimedChannel.channelClaim.constraint.scope.map((scope) => (
+            <Chip key={scope} variant="filled" color="primary" label={t(`registration.publication_types.${scope}`)} />
+          ))}
+        </StyledTableCell>
+        {isOnSettingsPage && (
+          <>
+            <StyledTableCell>
+              <Tooltip title={t('common.remove')}>
+                <IconButton
+                  data-testid={`delete-channel-claim-${channelIdentifier}`}
+                  onClick={() => setOpenConfirmDialog(true)}
+                  size="small"
+                  sx={{ width: '1.5rem', height: '1.5rem' }}>
+                  <CloseOutlinedIcon fontSize="inherit" />
+                </IconButton>
+              </Tooltip>
+            </StyledTableCell>
+          </>
         )}
-      </StyledTableCell>
-      <StyledTableCell aria-live="polite" aria-busy={organizationQuery.isPending}>
-        {organizationQuery.isPending ? (
-          <Skeleton width={300} />
-        ) : organizationName ? (
-          <Typography>{organizationName}</Typography>
-        ) : (
-          <Typography sx={{ fontStyle: 'italic' }}>{t('common.unknown')}</Typography>
-        )}
-      </StyledTableCell>
-      <StyledTableCell>
-        <StyledChip
-          variant="filled"
-          color="secondary"
-          size="small"
-          sx={{
-            bgcolor: publishingPolicy === 'Everyone' ? 'publishingRequest.main' : 'centralImport.main',
-          }}
-          label={t(`editor.institution.channel_claims.access_policies.${publishingPolicy}`)}
-          icon={
-            publishingPolicy === 'Everyone' ? <LockOpenIcon fontSize="small" /> : <LockOutlineIcon fontSize="small" />
-          }
-        />
-      </StyledTableCell>
-      <StyledTableCell>
-        <StyledChip
-          variant="filled"
-          color="secondary"
-          size="small"
-          sx={{
-            bgcolor: editingPolicy === 'Everyone' ? 'publishingRequest.main' : 'centralImport.main',
-          }}
-          label={t(`editor.institution.channel_claims.access_policies.${editingPolicy}`)}
-          icon={editingPolicy === 'Everyone' ? <LockOpenIcon fontSize="small" /> : <LockOutlineIcon fontSize="small" />}
+      </TableRow>
+
+      <ConfirmDialog
+        open={openConfirmDialog}
+        title={t('editor.institution.channel_claims.delete_channel_claim')}
+        isLoading={deleteMutation.isPending}
+        onAccept={async () => await deleteMutation.mutateAsync()}
```

Comment:
Her venter vi egentlig ikke på noe og all loading state ligger i `isPending` utenfor her. Derfor trenger vi egentlig ikke å bry oss om promise-håndtering her og kan forenkle litt så vidt jeg kan se
```suggestion
        onAccept={deleteMutation.mutate}
```